src/pages/BrowserPage/BrowserPage.js

```diff
@@ -9,11 +10,14 @@ const BrowserPage = () => {
   const navigate = useNavigate();
   const params = useParams();
   useEffect(() => {
-    fetch("/api/dishes")
-      .then((res) => res.json())
-      .then((json) => {
-        setDishes(json.dishes);
-      });
+    const fetchDishes = async () => {
+      const {data: res} = await axios(
+        "https://cookbook-backend-coderscamp.herokuapp.com/api/dishes"

```

Comment:
It would be a better idea to put base url `https://cookbook-backend-coderscamp.herokuapp.com/api/` into separate file. Axios let's you define custom axios instance and put there universal configuration for every endpoint. Create something like `axios.js` file and export the axios instance so you can use it everywhere.
Doc: https://axios-http.com/docs/instance